tests/test_client.py

```diff
@@ -132,32 +142,136 @@ def get_segments(self, advertisable_eid):
         response = self.get('advertisable/get_adgroups', data=data)
 
 
+    def create(self, stream):
+        if stream == 'advertisables':
+            raise Exception("Creating {} objects can cause exponential increase in api calls".format(stream))
+        elif stream == 'ads':
+            return self.create_ad()
+        elif stream == 'ad_reports':
+            raise NotImplementedError
+        elif stream == 'campaigns':
+            return self.create_campaign()
+        elif stream == 'segments':
+            return self.create_segment()
+        elif stream == 'ad_groups':
+           campaign_eid = random.choice(self.get_all_campaigns()).get('eid')
+           return self.create_ad_group(campaign_eid)
+        else:
+            raise NotImplementedError
+
+    def get_ad_file(self):
+        path = '/opt/code/tap-adroll/tests/stitch_loader.png'
```

Comment:
I think we want this to be more generic so we don't have to rely on a specific file system.

Reply:
fixed